[Support][FL] Update troubleshooting-cloudflare-5xx-errors.md (#12723)
cloudflare/cloudflare-docs: content/support/Troubleshooting/cloudflare-errors/troubleshooting-cloudflare-5xx-errors.md

> If the error does not mention “cloudflare,” contact your hosting provider for assistance on [502/504 errors from your origin](#502504-from-your-origin-web-server).

Reviewer: ```suggestion
If the error does not mention "cloudflare," contact your hosting provider for assistance on [502/504 errors from your origin](#502504-from-your-origin-web-server).
```

> If the error does not mention “cloudflare,” contact your hosting provider for assistance on [502/504 errors from your origin](#502504-from-your-origin-web-server).

Reviewer: ```suggestion
If the error does not mention "cloudflare," contact your hosting provider for assistance on [502/504 errors from your origin](#502504-from-your-origin-web-server).
```

> If the error does not mention “cloudflare,” contact your hosting provider for assistance on [502/504 errors from your origin](#502504-from-your-origin-web-server).

Reviewer 2: ```
If the error does not mention `cloudflare`, contact your hosting provider for assistance on [502/504 errors from your origin](#502504-from-your-origin-web-server).
```

After @Erisa teaching me why for grammatical correctness the comma should be in the quotes, a good compromise to avoid confusing the reader might be to use the code backticks instead. The reason it might be confusing to some readers is they may interpret us as asking them to look for literally cloudflare with a comma in their HTTP responses, which is not what we're asking for :) .

HTTP error 503 occurs when your origin web server is overloaded. There are two possible causes discernible by error message:

> -   Error doesn’t contain `cloudflare` or `cloudflare-nginx` in the HTML response body.

Reviewer: ```suggestion
-   Error doesn't contain `cloudflare` or `cloudflare-nginx` in the HTML response body.
```

Very small edit to the quote in `doesn't` which was using a mac "smart quote"
